apps/openmw/mwworld/esmstore.cpp

```diff
@@ -301,12 +302,14 @@ void ESMStore::setUp(bool validateRecords)
     if (validateRecords)
     {
         validate();
-        countRecords();
+        countAllCellrefs();
     }
 }
 
-void ESMStore::countRecords()
+void ESMStore::countAllCellrefs()
```

Comment:
countAllCell**R**efs

Because type is named `CellRef` https://github.com/OpenMW/openmw/blob/5ba3665c4221554cd823aedc789ce4c6af44eb10/components/esm/cellref.hpp#L42 . This will make searching easy.

Reply:
Good idea.